app/main.py

```diff
@@ -1,13 +1,20 @@
 def format_linter_error(error: dict) -> dict:
-    # white your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # white your code here
-    pass
+    return {"errors": [format_linter_error(error) for error in errors],
```

Comment:
please, make the code style as mentioned here:
https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md

`The curly braces must be located in one of two options: open and start with the text or have a line break between the text.`